contrib/docker/.env.template

```diff
@@ -42,4 +42,6 @@ POSTGRES_PORT=5432
 # Readwrite user/pass will be ckan:POSTGRES_PASSWORD
 # Readonly user/pass will be datastore_ro:DATASTORE_READONLY_PASSWORD
 DATASTORE_READONLY_PASSWORD=datastore
-
+#
+# Image: datapusher
+DATAPUSHER_VERSION=0.0.15
```

Comment:
In order to use docker-compose, we need to set the .env file. How about putting the datapuser version environment variable in the .env file?
The user may be able to change the datapusher version.